community-operators/synopsys/synopsys-operator.2019.1.0.yaml

```diff
@@ -0,0 +1,288 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: synopsys-operator.2019.1.0
+  namespace: placeholder
+  annotations:
+      categories: "Application Security, Monitoring"
```

Comment:
Be sure your categories match values given here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories

Reply:
fixed in latest push